ros/lib/kessel.py

```diff
@@ -0,0 +1,171 @@
+from typing import Optional
+from dataclasses import dataclass
+import grpc
+from kessel.inventory.v1beta2 import (
+    inventory_service_pb2_grpc,
+    streamed_list_objects_request_pb2,
+    representation_type_pb2,
+    subject_reference_pb2,
+    resource_reference_pb2,
+    reporter_reference_pb2,
+    request_pagination_pb2,
+    check_request_pb2,
+    allowed_pb2
+)
```

Comment:
Is there document and reference about these imports to understand,
1. When to use these with what combination?
2. What hierarchy to follow when using these and what is not allowed?
3. In what situation these will need change?